test/hotspot/jtreg/compiler/codecache/CodeCacheFullCountTest.java

```diff
@@ -57,7 +57,10 @@ public static void runTest() throws Throwable {
         ProcessBuilder pb = ProcessTools.createJavaProcessBuilder(
           "-XX:ReservedCodeCacheSize=2496k", "-XX:-UseCodeCacheFlushing", "-XX:-MethodFlushing", "CodeCacheFullCountTest", "WasteCodeCache");
         OutputAnalyzer oa = ProcessTools.executeProcess(pb);
-        oa.shouldHaveExitValue(0);
+        // Ignore adapter creation failures
+        if (oa.getExitValue() != 0 && !oa.getStderr().contains("Out of space in CodeCache for adapters")) {
+            throw new RuntimeException("VM finished with exit code " + oa.getExitValue());
```

Comment:
We also need `oa.reportDiagnosticSummary();` in this block. Otherwise we lose debugging information when throwing this exception.

Reply:
Thank you for your advice.
I have modified the point you had mentioned.

Thanks,
Kimura Yukihiro